### ISSUE TITLE
Multiple main.yml files

### ISSUE THREAD

**Issue by viveknath3222** [SamKirkland/FTP-Deploy-Action#414]
Use Case:

I've multiple branches -

1. Development - On my local I made changes to code and push to **development** branch and its not connected to any server.

2. Staging - This branch needs to connect to a staging server. When all the testing is done I'll merge **development** branch to **staging** branch.

3. Main - This branch needs to connect to main server. When all the testing is done I'll merge **staging** branch to **main** branch.

Is this posible?

**Comment by Trainax** [comment]
Yes, it is possible. Try taking a look at [this page](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions) which explains the workflow syntax.

I think you will be able to do what you want by using the

```
on:
  push:
    branches:
      - branch_name
```

filter
